specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -3380,6 +3397,248 @@
       },
       "description": "Describes the destination of connection monitor."
     },
+    "ConnectionMonitorEndpoint": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of connection monitor endpoint."
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "The ID of the resource used as an endpoint by connection monitor."
+        },
+        "address": {
+          "type": "string",
+          "description": "Address of connection monitor endpoint. Can be FQDN or IP address."
+        },
+        "filter": {
+          "$ref": "#/definitions/ConnectionMonitorEndpointFilter",
+          "description": "Additional information about connection monitor endpoint."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "Describes the connection monitor endpoint."
+    },
+    "ConnectionMonitorTestGroup": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of connection monitor test."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "A value indicating whether test is enabled."
```

Comment:
Same comment as above, shouldn't it be "test group" instead of "test"?

Reply:
Fixed